Makefile

```diff
@@ -29,7 +29,7 @@ sdk:
 	$(MAKE) -C sdk
 
 # Generate all test policy
-test_cases: sdk
+test_cases:
```

Comment:
When is this dependency firing? We have trustzone-test-collateral below. Does there ever need to be a test-collateral build without TEE set?